__tests__/util.test.ts

```diff
@@ -6,13 +6,26 @@ import expect from 'expect';
 
 // Module under test.
 import {
+  factToCondition,
   normalizeHex32,
 } from '../src/lib/util';
 
 const HEX_32_BYTES_LOWER_NO_PREFIX_RE = /^[0-9a-f]{64}$/;
 
 describe('util', () => {
 
+  describe('factToCondition()', () => {
+
+    it('succeeds', () => {
+      expect(factToCondition(
```

Comment:
just created this test using a random address/32byte hash found on mainnet, and then plugged into remix solidity editor
```
function testhash(address registry, bytes32 fact) external pure returns (bytes32) {
  return (keccak256(abi.encodePacked(registry, fact)));
}
```